src/features/goblins/bank/components/Deposit.tsx

```diff
@@ -1,48 +1,266 @@
+                            "text-shadow shadow-inner shadow-black bg-brown-200 w-full p-2",
+                            {
+                              "text-error": amountGreaterThanBalance,
+                            }
+                          )}
+                        />
+                        <span className="text-xxs md:text-xs absolute top-1/2 -translate-y-1/2 right-2">{`${
+                          isMobile ? "Bal" : "Balance"
+                        }: ${formattedSflBalance}`}</span>
+                      </div>
+                      <div className="w-[10%] flex self-center justify-center">
+                        <img className="w-6" src={token} alt="sfl token" />
+                      </div>
+                    </div>
+                  </>
+                )}
+                {hasItemsInInventory && (
+                  <>
+                    <p className="text-sm">Collectibles</p>
+                    <div className="flex flex-wrap h-fit -ml-1.5">
+                      {depositableItems.map((item) => {
+                        return (
+                          <Box
+                            count={inventoryBalance[item]}
+                            key={item}
+                            onClick={() => onAddItem(item)}
+                            image={ITEM_DETAILS[item].image}
+                            canBeLongPressed
+                          />
+                        );
+                      })}
+                    </div>
+                  </>
+                )}
+              </div>
+              <div className="pt-3">
+                <p className="mb-1">Your farm will receive:</p>
+                <div className="text-[11px] sm:text-xs mb-3">
+                  <CopyAddress address={farmAddress} />
+                </div>
+                <div className="space-y-3">
+                  {validDepositAmount && <p>{sflDepositAmount} SFL</p>}
+                  {hasItemsToDeposit && (
+                    <div className="flex flex-wrap h-fit -ml-1.5">
+                      {selectedItems.map((item) => {
+                        return (
+                          <Box
+                            count={inventoryToDeposit[item]}
+                            key={item}
+                            onClick={() => onRemoveItem(item)}
+                            image={ITEM_DETAILS[item].image}
+                            canBeLongPressed
+                          />
+                        );
+                      })}
+                    </div>
+                  )}
+                </div>
+              </div>
+            </div>
+            <div className="mb-1 mt-2">
+              <a
+                target="_blank"
+                className="underline text-xxs hover:text-blue-500"
+                href={`https://app.gitbook.com/o/bntEYvEP4dzoxNUbvnHA/s/IWGEodCG0c07OUY9iW9t/~/changes/297/economy/depositing-and-custody#cant-see-the-items-you-deposited`}
```

Comment:
Just a reminder to update this to the docs.sunflower-land.com link before merging ;)

Reply:
Already done mate.